examples/api/index.ts

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 import * as aws from "@pulumi/aws";
+import { x } from "@pulumi/aws/apigateway";
```

Comment:
Nit - but this feels a little odd - `x.API` isn't particularly descriptive.

Reply:
fair enough.  :)

Reply:
done.